src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -283,7 +283,7 @@ private static function getClassHierarchy(\ReflectionClass $class)
 
         $traits = array();
 
-        if (function_exists('get_declared_traits')) {
+        if (method_exists('ReflectionClass', 'getTraits')) {
```

Comment:
not related but still worth cleaning: `get_declared_traits` is not used after this check; `ReflectionClass::getTraits` is.